src/PrestaShopBundle/Form/Admin/Type/TextPreviewType.php

```diff
@@ -64,20 +65,27 @@ public function getBlockPrefix()
         return 'text_preview';
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function configureOptions(OptionsResolver $resolver)
     {
         parent::configureOptions($resolver);
 
         $resolver
             ->setRequired(['preview_class'])
+            ->setDefaults(
+                [
+                    'preview_class' => 'text-preview',
+                    'prefix' => null,
+                    'suffix' => null,
+                    'required' => false,
+                    'error_bubbling' => true,
+                    'compound' => false,
+                    'allow_html' => false, // Start automatically TinyMCE
```

Comment:
Again with this weird comment 😅
And I don't understand the need of all those new options which are not even used?

Reply:
Yeah looked over and I don't think they are needed, they are the same in hidden type so should be succesfully inherited from it.